custom_functions/mark_evidence.py

```diff
@@ -49,29 +37,91 @@ def mark_evidence(container=None, object_id=None, content_type=None, **kwargs):
     else:
         raise TypeError("The input 'container' is neither a container dictionary nor an int, so it cannot be used")
     
-    # Build json dictionary for requests post
-    data = {
-        "container_id": container_id,
-        "object_id": object_id,
-        "content_type": content_type,
-        }
+    # If content added is type 'action run',
+    # then iterate through an input object that is a results object,
+    # and append the action_run_id's to data
+    if isinstance(input_object, list) and content_type.lower() == 'actionrun_id':
+        for action_result in input_object:
+            if action_result.get('action_run_id'):
```

Comment:
Lines 45 and 48 are inconsistent now, with one being action_run_id and one being actionrun_id. It looks like action_run_id is the correct one so line 48 should be changed.